packages/vaadin-avatar/src/vaadin-avatar.js

```diff
@@ -330,6 +336,15 @@ class AvatarElement extends ElementMixin(ThemableMixin(PolymerElement)) {
       this.setAttribute('title', this.i18n.anonymous);
     }
   }
+
+  /** @private */
+  __handleImageLoadError() {
```

Comment:
Taking a look over the other components, I found out there is generally being used the `on` prefix for event methods.

https://github.com/vaadin/web-components/blob/master/packages/vaadin-avatar/src/vaadin-avatar-group.js#L340

https://github.com/vaadin/web-components/blob/master/packages/vaadin-context-menu/src/vaadin-context-menu.js#L431

Reply:
Makes sense, done